### PR TITLE
Add tooltip information to Seismic Reader

### PR DESCRIPTION
**Pull request by thommy101**
Added some information about usage to the Seismic Reader, almost the same as the Seismic Vibrator.
Adds feature #4743

Opened as PR to make sure the used wording for the tooltip is right ;)

### CODE REVIEW

File: src/main/resources/assets/mekanism/lang/en_us.lang [hunk 1]
```diff
@@ -805,6 +805,7 @@ tooltip.flamethrower.modeBump=Flamethrower mode bumped to
 
 tooltip.seismicReader.needsEnergy=Not enough energy to interpret vibration
 tooltip.seismicReader.noVibrations=Unable to discover any vibrations
+tooltip.SeismicReader=A portable machine that uses seismic vibrations to provide information on differing layers of the world.
```
Comment (thiakil) [inline src/main/resources/assets/mekanism/lang/en_us.lang]: English is fine, but while we're at it, shouldn't we make it `tooltip.mekanism.SeismicReader`?

Comment (thommy101) [reply src/main/resources/assets/mekanism/lang/en_us.lang]: I've got a separate branch which will update all translations at once ;) Will update that later today

Comment (thiakil) [reply src/main/resources/assets/mekanism/lang/en_us.lang]: I know, but would it not be easier to add the new one with the new schema already?

Comment (thommy101) [reply src/main/resources/assets/mekanism/lang/en_us.lang]: Just looked, I apparently have updated tooltips already.. Will change this here ;)

**Pull request merged**